lnrpc/rpc.proto

```diff
@@ -503,12 +503,15 @@ service Lightning {
     }
 
     /** lncli: `sendpayment`
-    SendPayment dispatches a bi-directional streaming RPC for sending payments
-    through the Lightning Network. A single RPC invocation creates a persistent
-    bi-directional stream allowing clients to rapidly send payments through the
-    Lightning Network with a single persistent connection.
-    */
-    rpc SendPayment (stream SendRequest) returns (stream SendResponse);
+    Deprecated, use routerrpc.SendPayment. SendPayment dispatches a
+    bi-directional streaming RPC for sending payments through the Lightning
+    Network. A single RPC invocation creates a persistent bi-directional
+    stream allowing clients to rapidly send payments through the Lightning
+    Network with a single persistent connection.
+    */
+    rpc SendPayment (stream SendRequest) returns (stream SendResponse) {
+        option deprecated = true;
```

Comment:
does this mean the API is still available but users get a notice?

Reply:
It is only documentation

Reply:
As clarified offline: the API is still available, but users need to prep for its removal.